ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -619,4 +623,15 @@ static void maybeCreateProcessingLogStream(
 
     commandQueue.enqueueCommand(configured.get());
   }
+
+  private static void maybeConfigureRocksDBConfigSetter(final KsqlConfig ksqlConfig) {
```

Comment:
I'd be tempted to pull this out into a new class, and just call it from here.  If you inject the handler (in a `@VisableForTesting` constructor only), then the `KsqlRestApplicationTest` only needs to test the handler is invoked.  All the other tests can be in the handler test class.

Reply:
Note: The suggested `RocksDbConfigSetterWrapper` achieves the same.